### PR TITLE
Update ValkeyModuleEvent_MasterLinkChange to ValkeyModuleEvent_PrimaryLinkChange

### LINKED ISSUE #249: Usage of MASTER in module reference
We still have a usage of master in modules, https://github.com/valkey-io/valkey/blob/717ec1e1440d94ef1ebe064addaffc4970d88c29/src/module.c#L11514, even though we attempted to remove it. Was this a miss @PingXie, or is there some reason we have to keep it?

### PR DESCRIPTION
**Pull request by 9bany**
fixes: https://github.com/valkey-io/valkey/issues/249

### CODE REVIEW

File: src/module.c [hunk 1]
```diff
@@ -11511,18 +11511,6 @@ static uint64_t moduleEventVersions[] = {
  *
  *         int32_t hz;  // Approximate number of events per second.
  *
- * * ValkeyModuleEvent_MasterLinkChange
```
Comment (madolson) [inline src/module.c:11514]: Instead of deleting this, can we just update it to `ValkeyModuleEvent_PrimaryLinkChange` ?

Comment (zuiderkwast) [reply src/module.c:11514]: yeah we need to keep it, just rename it.

@madolson be careful when you say "remove" next time. :laughing:

Comment (9bany) [reply src/module.c:11514]: Oke, Well done
I renamed it to `ValkeyModuleEvent_PrimaryLinkChange`

**Review by madolson — APPROVED** [review]

**Comment by PingXie** [comment]
I think the Redis event name needs to be reverted back to Master to maintain source compat. Changing `Master` to `Primary` is good for the Valkey macro.

`RedisModuleEvent_MasterLinkChange`

**Comment by madolson** [comment]
Yes, let's fix this for RC2. We'll see if anyone comments about it.

**Comment by madolson** [comment]
@PingXie Fix in https://github.com/valkey-io/valkey/pull/289

**Pull request merged**